src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -211,6 +211,7 @@ LedgerMaster::setValidLedger(
             !app_.getMaxDisallowedLedger() ||
             l->info().seq + max_ledger_difference_ >
                     app_.getMaxDisallowedLedger());
+    (void) max_ledger_difference_;
```

Comment:
I think we do that enough we don't need a comment. `(void)some_var` is clearly to resolve a warning. I'd rather not draw attention to something so unimportant with a comment.